Packet++/src/TcpLayer.cpp

```diff
@@ -118,11 +133,11 @@ TcpOption TcpLayer::addTcpOption(const TcpOptionBuilder& optionBuilder)
 	return addTcpOptionAt(optionBuilder, getHeaderLen()-m_NumOfTrailingBytes);
 }
 
-TcpOption TcpLayer::addTcpOptionAfter(const TcpOptionBuilder& optionBuilder, TcpOptionType prevOptionType)
+TcpOption TcpLayer::insertTcpOptionAfter(const TcpOptionBuilder& optionBuilder, TcpOptionEnumType prevOptionType)
```

Comment:
`const TcpOptionEnumType prevOptionType`

Reply:
same for all others

Reply:
not sure if we need to change the function name, since you can use overloading here to keep  the original name.

Reply:
Unfortunately we cannot use name overloading in this case because TcpOptionType parameter has a default value. Compiler says it's an ambiguous call